packages/www/components/Site/Pricing/faq.md

```diff
@@ -44,9 +44,8 @@ a Hacker account, your account will be disabled until you select a paid plan.
 ### **What currencies does Livepeer Studio accept?**
 
 USD is the only currency accepted right now via the Livepeer Studio dashboard.
-Crypto-native payments are coming later this year. If you prefer to pay with
-cryptocurrency now, you may reach out
-at [help@livepeer.org](mailto:help@livepeer.org)
+Crypto-native payments are coming soon. If you prefer to pay with cryptocurrency
```

Comment:
Can we fix this as well while at it https://discord.com/channels/423160867534929930/611650333243867170/1187168731504320562